packages/js/components/src/phone-number-input/index.tsx

```diff
@@ -0,0 +1,219 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useRef, useLayoutEffect } from 'react';
```

Comment:
Why are the hooks being imported from the react package instead of `@wordpress/element`? This question applies for elsewhere in this changeset as well.

Reply:
Thanks for catching it, I'm just used to importing them directly from `react` in WooPayments and forgot about using `@wordpress/element` here 🙂 Addressed in 4294906